pkgs/development/python-modules/orbax-checkpoint/default.nix

```diff
@@ -0,0 +1,78 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, flit-core
+, absl-py
+, etils
+, importlib-resources
+, jax
+, jaxlib
+, msgpack
+, nest-asyncio
+, numpy
+, protobuf
+, pyyaml
+, tensorstore
+, typing-extensions
+, flax
+, pytest
+, pytest-xdist
+, pytestCheckHook
+}:
+
+let self = buildPythonPackage rec {
+  pname = "orbax-checkpoint";
+  version = "0.4.2";
+  pyproject = true;
+
+  disabled = pythonOlder "3.9";
+
+  src = fetchPypi {
+    pname = "orbax_checkpoint";
+    inherit version;
+    hash = "sha256-UaIfClZ76noFc5BUoCnMBtZ3PuRA3G5TCQHSI0KD0NI=";
+  };
+
+  nativeBuildInputs = [
+    flit-core
+  ];
+
+  propagatedBuildInputs = [
+    absl-py
+    etils
+    jax
+    jaxlib
+    importlib-resources
+    msgpack
+    nest-asyncio
+    numpy
+    protobuf
+    pyyaml
+    tensorstore
+    typing-extensions
+  ];
+
+  nativeCheckInputs = [
+    flax
+    pytest
+    pytest-xdist
+    pytestCheckHook
+  ];
+
+  doCheck = false;
+
+  passthru.tests = {
+    check = self.overridePythonAttrs (_: { doCheck = true; });
+  };
+
+  pythonImportsCheck = [ "orbax.checkpoint" ];
+
+  meta = with lib; {
+    description = "Orbax Checkpoint";
```

Comment:
`orbax-checkpoint` is `Orbax Checkpoint`, that's a nice description.😵‍💫

How about citing `A checkpointing library oriented towards JAX user`?
https://github.com/google/orbax/blob/8ab806ce17186a5f0700a52aa089d8317d8d03bf/README.md?plain=1#L16

Reply:
Definitely, I changed it.